ccloud/beginner-cloud/start.sh

```diff
@@ -271,7 +270,7 @@ ccloud kafka acl delete --allow --service-account $SERVICE_ACCOUNT_ID --operatio
 ##################################################
 
 TOPIC3="demo-topic-3"
-
+CONNECTOR="datagen_ccloud_pageviews"
```

Comment:
nit alert:  WDYT about coordinating the file name and the connector name?  The terms are ordered differently.

Reply:
Hehe I like the "nit alert" callout, I have edited the instructions, start script, and the json config file name to all match the connector name.